Assets/Plugins/LeapMotion/Core/Scripts/Query/QueryOperatorExtensions.cs

```diff
@@ -19,6 +19,34 @@ namespace Leap.Unity.Query {
   /// </summary>
   public static class QueryOperatorExtensions {
 
+    /// <summary> Don't put big r values into this! </summary>
+    private static int getCombinationLength(int n, int r) {
+      if (r > 8) throw new System.NotSupportedException();
+      var result = 1;
+      var rFac = Factorial(r);
+      for (var i = 0; i < r; i++) result *= n - i;
+      result /= rFac;
+      return result;
+    }
+    /// <summary> Int32s, careful with overflow. </summary>
+    private static int Factorial(int N) {
+      int f = 1; for (var n = N; n > 1; n--) f *= n; return f;
+    }
+
+    /// <summary>
+    /// Prints a Debug.LogError if the query results contains more than a single element, but otherwise preserves the Query as-is.</summary>
+    public static Query<T> ComplainIfMany<T>(this Query<T> query, string message = null) {
```

Comment:
weird <summary></summary> style here (recommend sandwich both horizontally on the same line or vertically on distinct lines)
```suggestion
    /// <summary>
    /// Prints a Debug.LogError if the query results contains more than a single element, but otherwise preserves the Query as-is.
    /// </summary>
    public static Query<T> ComplainIfMany<T>(this Query<T> query, string message = null) {
```